### ISSUE TITLE
Point NPM link to module

### ISSUE THREAD

**Issue by mingard** [dadi/docs#31]
Currently the *NPM* Link [here](http://docs.dadi.tech/cdn/getting-started/installing/) points to NPM root. Could it point to the modules page?

**Comment by jimlambie** [comment]
You're welcome to edit it

**Comment by mingard** [comment]
@jimlambie great

**Comment by abovebored** [comment]
> All our platform microservices are available from NPM. 

The thinking was with that line to link people who might have never heard of NPM to the homepage so they can learn more about it (we link to the exact page elsewhere) - I can see why it would make sense to change, but just FYI.

**Comment by mingard** [comment]
@abovebored that makes sense. I'll reword it to include both!
